modules/storages/lib/api/v3/file_links/file_link_collection_representer.rb

```diff
@@ -29,7 +29,7 @@
 module API
   module V3
     module FileLinks
-      class FileLinkCollectionRepresenter < ::API::Decorators::OffsetPaginatedCollection
+      class FileLinkCollectionRepresenter < ::API::Decorators::UnpaginatedCollection
```

Comment:
Because Paginated expects am ActiveRecord relationship object, and uses it to guess the model class and some ordering/page information, and it fails because the code only provides an Array. Switching to Unpaginated avoids this.

We may have to reconsider this choice in the future. Again, this is the minimal amount of code to make the spec pass.

Reply:
Gotcha

Reply:
I believe that for the purpose it is used in, using an unpaginated response is correct. It is used to return all the `FileLinks` created and the client will never be able to paginate through that response. So in case the client passes thousands of `FileLinks` at a time, that is what we will have to return. 

Though thinking about this, do we somehow limit the amount of `FileLinks` that can be created at a time?

Reply:
No there is no limits about the number of `FileLink`s that can be created at a time. What would be a sensible default?

Reply:
Set 20 at first, and see if anyone complains. + Document it in api docs